docs/content/concepts/repositories-workspaces/repositories.mdx

```diff
@@ -83,7 +101,7 @@ If you save the code above as `repo.py`, you can then run the Dagster command li
 dagit -f repo.py
 ```
 
-Now you can see that all jobs in this repository are on the left:
+Now you can see that all the assets and jobs in this repository are listed in the left sidebar. Assets are organized in groups. In our example, `asset1` and `asset2` are placed in the `default` group (because they were not explicitly assigned a group). `asset3` is in `mygroup`.
```

Comment:
```suggestion
Now you can see that all the assets and jobs in this repository are listed in the left sidebar. Assets are organized in groups. In our example, `asset1` and `asset2` are placed in the `default` group because they were not explicitly assigned a group. `asset3` is in `mygroup`.
```

Not sure we need the parentheses here - I found it a bit distracting, so suggesting we remove it.